backend/api/src/model/event.rs

```diff
@@ -17,7 +17,7 @@ pub(crate) struct Event {
     updated: DateTime<Utc>,
     creator: Option<String>,
 
-    thumbnail: String,
+    thumbnail: Option<String>,
```

Comment:
The commit message talks about `resolution`, but the changeset is about `thumbnails`. :thinking: